ui/src/app/base/components/ActionButton/ActionButton.md

```diff
@@ -0,0 +1,21 @@
+### Default
+
+```jsx
+<ActionButton appearance="positive">Click me!</ActionButton>
+```
+
+### Loading
+
+```jsx
+<ActionButton appearance="positive" loading>
+  Click me!
+</ActionButton>
+```
+
+### Success
+
+```jsx
+<ActionButton appearance="positive" success>
```

Comment:
This isn't working because the internal logic requires that the button moves from loading to success. Do you know if there's a way to have a "dynamic" example that can show these state transitions? Otherwise you can probably remove this section.

Reply:
I've added a second button to transition the state when clicked.